src/containers/Dashboard.vue

```diff
@@ -76,6 +76,31 @@ import surfing from '@/assets/images/surfing.svg';
 
 import store, { initializeFirestoreListeners } from '@/store';
 import { immutableBottomBarState } from '@/components/BottomBar/BottomBarState';
+import {
+  smallBreakpoint,
+  mediumBreakpoint,
+  veryLargeBreakpoint,
+} from '@/assets/scss/_variables.scss';
+
+const smallBreakpointPixels = parseInt(
+  smallBreakpoint.substring(0, smallBreakpoint.length - 2),
+  10
+);
+const mediumBreakpointPixels = parseInt(
+  mediumBreakpoint.substring(0, mediumBreakpoint.length - 2),
+  10
+);
+const veryLargeBreakpointPixels = parseInt(
+  veryLargeBreakpoint.substring(0, veryLargeBreakpoint.length - 2),
+  10
+);
+
+const getMaxButtonBarTabs = () => {
+  if (window.innerWidth <= smallBreakpointPixels) {
+    return window.innerWidth <= veryLargeBreakpointPixels ? 1 : 2;
+  }
+  return 4;
+};
```

Comment:
Wouldn't we want to swap for `veryLargeBreakpointPixels` and `smallBreakpointPixels` here?

The behavior is a little wonky:

https://user-images.githubusercontent.com/54298311/111017380-2f266f80-8381-11eb-9b22-4b1f4860e81e.mov

Reply:
@tcho6319 fixed now